pkg/tests/build_test.sh

```diff
@@ -55,6 +55,15 @@ function get_tar_permission() {
   tar tvf "${test_data}" | fgrep "00 $file" | cut -d " " -f 1
 }
 
+function get_zip_listing() {
+  local input=$1
+  local test_data="${TEST_DATA_DIR}/${input}"
+  # Remove table headers, summary and filter to only the columns we care about
+  unzip -v "${test_data}" 2>/dev/null \
+    | awk 'NR>3 && $1 == "--------" {exit} NR>3 {print $1, $2, $5, $6, $7, $8}' \
+    | sed 's:01-01-1980:1980-01-01:g' # Normalize dates
```

Comment:
I'm open to alternatives :) My Mac did this differently from my Windows/Linux builds. I haven't investigated it. 🤷‍♂

Reply:
@j3parker consider using `zipinfo` with `-T` switch; from `zipinfo`'s help: `-T  print file times in sortable decimal format`. I've checked in on macOS and Linux and it produces consistent datetime (`19800101.000000`)

Reply:
My preference is no more shell tests.

This could be a distinct python test that used python's zlib apis to unzip and see that the archive structure matched what you expect.

Reply:
I've removed the shell tests